AutoscoperM/AutoscoperM.py

```diff
@@ -366,10 +364,7 @@ def stopAutoscoper(self, force=True):
         if self.AutoscoperSocket:
             self.disconnectFromAutoscoper()
 
-        if force:
-            self.AutoscoperProcess.kill()
-        else:
-            self.AutoscoperProcess.terminate()
+        self.AutoscoperProcess.kill()
```

Comment:
Assuming this change and the one above checking if the connection is already established are improvement, they should be done in a dedicated commit.

If the changes are part of the solution, the commit message should explain it.

Reply:
> Fixed bug with AutoscoperM refusing to reconnect to Autoscoper server if the connection was severed outside of the Slicer environment (ie. Autoscoper crashed and you tried to relaunch it from Slicer)

In other words, this should be fixed in its own commit.